File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''
    result = []
    s = s.lower()
    words = list(s.split(" "))
    
    for char in words:
        if char != "":
            result.append(char[0].upper() + char[1:])
        else:
            result.append("")


    answer = ' '.join(result)
    return answer

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    let answer = '';
    const arr = s.toLowerCase().split('');
    arr.forEach((val, index) => {
        if(arr[index - 1] == ' ' || index == 0){
            answer += val.toUpperCase();
        }else{
            answer += val.toLowerCase();
        }
    })
    return answer;
}